ast/functions.go

```diff
@@ -364,6 +379,11 @@ type AggregateFuncExpr struct {
 	// For example, column c1 values are "1", "2", "2",  "sum(c1)" is "5",
 	// but "sum(distinct c1)" is "3".
 	Distinct bool
+
+	CurrentGroup string
```

Comment:
When call update or evaluate, we should specify the group key by setting this attribute.